evasquez26/angular4-startup
# S5-ONE-52 Join course as teacher

## Story / Task

see [ONE-52](https://bixlabs.atlassian.net/browse/ONE-52)


## What this PR does?

Adds functionality to request to join a course as teacher

### Before 

You could not request to be a teacher of a given course

### After

You can request to be a teacher of a course

### How to test / reproduce

- Login to the app.
- Go to schools, select a school where you are an admin or teacher, and go to courses.
- Select a course you're not the owner
- See there's a button to join the course as teacher
- Click and confirm
- You should see the button changed to requested.

### TODOs
- [x] Rebase after merging #47 

### Related PRs
N/A

## Migrations
NO


Thanks.
